### ISSUE TITLE
Do you support Linux version?

### ISSUE THREAD

**Issue by eunseop90** [POSTECH-IMLAB/PIMNet_FR-1#4]
Hi

I want to use it on Linux since my computer is based on Linux.

So do you plan to support or support Linux version?

**Comment by bnkang** [comment]
@eunseoplee90 Yes, I tested this program is working well on linux without GUI components.
We will schedule to release full source files without GUI soon.
